os/net/routing/rpl-lite/rpl-neighbor.c

```diff
@@ -378,13 +361,11 @@ best_parent(int fresh_only)
     }
 
 #if UIP_ND6_SEND_NS
-    {
     uip_ds6_nbr_t *ds6_nbr = rpl_get_ds6_nbr(nbr);
     /* Exclude links to a neighbor that is not reachable at a NUD level */
-    if(ds6_nbr == NULL || ds6_nbr->state != NBR_REACHABLE) {
+    if(ds6_nbr == NULL) {
```

Comment:
Would it even make sense to skip the variable and just check the function return value directly?
      if(rpl_get_ds6_nbr(nbr) == NULL)
?